docs/checks/tollValidationCheck

```diff
@@ -0,0 +1,41 @@
+# TollValidationCheck
+
+#### Description
+
+The purpose of this check is to identify ways that need to have their toll tags investigated/added/removed.
+
```

Comment:
It would be nice to have a description of the configurables here.

Reply:
addressed and fixed!